packages/search/src/ago/compute-items-facets.ts

```diff
@@ -0,0 +1,79 @@
+import { UserSession } from "@esri/arcgis-rest-auth";
+import { agoSearch } from "./search";
+import { ISearchParams } from "./params";
+import { getProp } from "@esri/hub-common";
+import { format, hasApi, collection, downloadable } from "./helpers/aggs";
+
+// these custom aggs are based on a field that are not supported by AGO aggregations
+const customAggsNotSupportedByAgo = ["downloadable"];
+
+// these custom aggs are based on a field that are supported by AGO aggregations
+const customAggsSupportedByAgo = ["hasApi", "collection"];
+
+const customAggsFunctions: { [key: string]: any } = {
+  downloadable,
+  hasApi,
+  collection
+};
+
+/*
+ * This Util function takes AGO results and formats them into an aggregations object
+ * that looks like it came from the V3 API. Many facets can be fairly easily mocked
+ * using a standardized approach (the 'else if' and 'else' branches) but a few facets
+ * require more complex and customized logic. Those go in the `customFacets` hash,
+ * where the name of the key is the name of the facet being computed and the custom function
+ * is implemented below.
+ */
```

Comment:
this code comment isn't being picked up by TSDoc.

<img width="1019" alt="Screen Shot 2019-05-01 at 9 20 25 AM" src="https://user-images.githubusercontent.com/3011734/57027864-69098500-6bf2-11e9-9b07-8dad4a3779ff.png">

for an example of method documentation that's formatted for display in our documentation, see: https://github.com/Esri/hub.js/blob/b336f324b1639993309ead0c87f91b13f737d97e/packages/annotations/src/add.ts#L29-L50

Reply:
nice! I will fix that